octoprint_discordremote/command.py

```diff
@@ -476,14 +478,139 @@ def unzip(self, params):
                         with zip.open(f) as source:
                             file.write(source.read())
 
-                self.plugin.get_file_manager().remove_file('local', unzippable.rpartition('/')[2])
+                self.plugin.get_file_manager().remove_file('local', unzippable)
 
         except:
-            return None, error_embed(author=self.plugin.get_printer_name(), title="Bad zip file.",
-                                 description='In case of multi-volume files, one could be missing.')
+            self.plugin.get_file_manager().remove_file('local', unzippable)
```

Comment:
doesn't this line delete the *.zip.001 file if there is a file missing? Thats probably a bit counter intuitive?

Reply:
The function tries to concatenate all .zip.001, ... files to one large .zip which is then treated the same as any regular zip uploaded by the user. When unzipping that large zip fails that zip is deleted, the .zip.001 files aren't touched after concatenating them. I'll double check that, though.

Reply:
I have checked and while this line doesn't remove any of the *.zip.XXX volumes, line 443/444 absolutely does, even when there are still volumes missing. I'll fix that.